packages/core/src/domain/patient-settings.ts

```diff
@@ -15,3 +16,11 @@ export interface PatientSettingsCreate extends BaseDomainCreate {
 }
 
 export interface PatientSettings extends BaseDomain, PatientSettingsCreate {}
+
+export type AdtSubscriberData = {
+  id: string;
+  cxId: string;
+} & Pick<
+  PatientData,
+  "firstName" | "lastName" | "dob" | "address" | "personalIdentifiers" | "genderAtBirth"
+>;
```

Comment:
I know we call the other type PatientData, but I don't think we need to propagate that sus naming choice elsewhere in the codebase. This type represents an AdtSubscriber / AdtEnrollee, we don't need data on the name.

Reply:
Related, in the sense it could drive an update to the name to be just `Subscriber`: https://github.com/metriport/metriport/pull/3464#discussion_r2008156439